src/test/java/eu/dissco/core/digitalspecimenprocessor/utils/TestUtils.java

```diff
@@ -9,12 +9,11 @@
 import eu.dissco.core.digitalspecimenprocessor.domain.DigitalSpecimenEvent;
 import eu.dissco.core.digitalspecimenprocessor.domain.DigitalSpecimenRecord;
 import eu.dissco.core.digitalspecimenprocessor.domain.HandleAttribute;
-import org.springframework.core.io.ClassPathResource;
-
 import java.io.IOException;
 import java.nio.charset.StandardCharsets;
 import java.time.Instant;
 import java.util.List;
+import org.springframework.core.io.ClassPathResource;
```

Comment:
Don't think this is necessary

Reply:
It's used for LoadResourceFile (line 197)